pkg/e2e/e2e.go

```diff
@@ -98,6 +98,16 @@ var _ = ginkgo.BeforeEach(func() {
 func beforeSuite() bool {
 	// Skip provisioning if we already have a kubeconfig
 	var err error
+	kubeconfigPath := viper.GetString(config.Kubeconfig.Path)
+	if kubeconfigPath != "" {
+		kubeconfigBytes, err := ioutil.ReadFile(kubeconfigPath)
+		if err != nil {
+			log.Printf("failed reading '%s' which has been set as the TEST_KUBECONFIG: %v", kubeconfigPath, err)
+			return false
+		}
+		viper.Set(config.Kubeconfig.Contents, string(kubeconfigBytes))
```

Comment:
Unless I'm missing something, this gets unconditionally overwritten here: https://github.com/openshift/osde2e/pull/848/files#diff-9ff84f56455e33400faa6125ff49618962967add5776347c6bacdde30bf3a5e6R188 (line 188)

Reply:
100% correct, will address